packages/screens/DAppStore/query/getFromFile.ts

```diff
@@ -164,14 +162,14 @@ export function getAvailableApps(): dAppGroup {
           description: "Enter the Bar, play games. punks!",
           route: "ToriPunks",
           groupKey: "top-apps",
-          selectedByDefault: false,
+          selectedByDefault: true,
           alwaysOn: false,
         },
       },
     },
-    "coming-soon": {
-      id: "coming-soon",
-      groupName: "Coming soon dApps",
+    bookmarks: {
+      id: "bookmarks",
+      groupName: "External Apps",
       icon: otherAppsIcon,
```

Comment:
The icon for "coming-soon" and  "bookmarks" doesn't appear 
![image](https://github.com/TERITORI/teritori-dapp/assets/50441633/85c47e53-9673-497b-b138-3fd759dda4b8)